### PR TITLE
Changes the return type from void as suggested in issue #3001

### LINKED ISSUE #3001: Content update: performance.mark|measure x UserTiming Level 3
## What page(s) did you find the problem on?

https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark
https://developer.mozilla.org/en-US/docs/Web/API/Performance/measure

## Specific page section or heading?

- Return value
- Specifications
- parameters

## What is the problem?

- New version of spec (UT L3) was released: https://www.w3.org/TR/user-timing-3/
- Shipped in [Chrome 78](https://www.chromestatus.com/feature/5149401886490624) and will ship soon in [Safari 14.1](https://webkit.org/blog/11333/release-notes-for-safari-technology-preview-115/). [Not yet in Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1581705)
- It makes `performance.mark|measure` return `PerformanceMark` `PerformanceMeasure` (and not `void`)

image

- `perf.mark` accepts  `startTime` param: 

```
      performance.mark('a', {startTime: 900})
      performance.mark('b', {startTime: 999})
      performance.measure('a->b', 'a', 'b')
```

image

### PR DESCRIPTION
**Pull request by himanshugarg**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'
Fixes #3001

> What was wrong/why is this fix needed? (quick summary only)
The return type of two functions `Performance.mark()` and `Performance.measure()` was void. Updated to suggested.

> Anything else that could help us review it
The bug asks for updating the parameter types as well. I have left those alone as Rumyra suggests updating return values, perhaps for a reason. And this had a '10-minute' label. But let me know if I should make further changes. Also note that the code examples are untouched.

### CODE REVIEW

File: files/en-us/web/api/performance/mark/index.html [hunk 1]
```diff
@@ -43,8 +43,8 @@ <h3 id="Arguments">Arguments</h3>
 <h3 id="Return_value">Return value</h3>
 
 <dl>
-  <dt>void</dt>
-  <dd></dd>
+  <dt>entry</dt>
+  <dd>The created {{domxref("PerformanceMark")}} entry</dd>
```
Comment (wbamberg) [inline files/en-us/web/api/performance/mark/index.html:47]:
```suggestion
  <dd>The {{domxref("PerformanceMark")}} entry that was created.</dd>
```

File: files/en-us/web/api/performance/measure/index.html [hunk 2]
```diff
@@ -65,8 +65,8 @@ <h3 id="Arguments">Arguments</h3>
 <h3 id="Return_value">Return value</h3>
 
 <dl>
-  <dt>void</dt>
-  <dd></dd>
+  <dt>entry</dt>
+  <dd>The created {{domxref("PerformanceMeasure")}} entry</dd>
```
Comment (wbamberg) [inline files/en-us/web/api/performance/measure/index.html:69]:
```suggestion
  <dd>The {{domxref("PerformanceMeasure")}} entry that was created.</dd>
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for the PR! I had a couple of wording suggestions.

I think "The created *thing*" is a bit clumsy, and we should have a period at the end of the sentence.

**Comment by himanshugarg** [comment]
Thanks for the suggestions. Committed.

**Review by wbamberg — APPROVED** [review]
👍 thanks for your contribution, @himanshugarg !

**Pull request merged**